modules/fuzztest/20240813.0/MODULE.bazel

```diff
@@ -0,0 +1,63 @@
+module(
+    name = "fuzztest",
+    version = "20240813.0",
+    repo_name = "com_google_fuzztest",
+)
+
+bazel_dep(
+    name = "abseil-cpp",
+    version = "20240116.1",
+    repo_name = "com_google_absl",
+)
+
+bazel_dep(
+    name = "re2",
+    version = "2024-04-01",
+    repo_name = "com_googlesource_code_re2",
+)
+
+bazel_dep(
+    name = "bazel_skylib",
+    version = "1.7.1",
+)
+
+# Dev dependencies.
+bazel_dep(
+    name = "googletest",
+    version = "1.14.0.bcr.1",
+    dev_dependency = True,
+    repo_name = "com_google_googletest",
+)
+
+bazel_dep(
+    name = "nlohmann_json",
+    version = "3.11.3",
+    dev_dependency = True,
+    repo_name = "com_nlohmann_json",
+)
+
+bazel_dep(
+    name = "protobuf",
+    version = "26.0",
+    dev_dependency = True,
+    repo_name = "com_google_protobuf",
+)
+
+bazel_dep(
+    name = "rules_proto",
+    version = "5.3.0-21.7",
+)
+
+bazel_dep(
+    name = "riegeli",
+    version = "0.0.0-20240606-973b6f0",
+    dev_dependency = True,
```

Comment:
This seems to be a required dependency for the two test targets as well.

Reply:
Turned off dev_dependency